src/olympia/devhub/templates/devhub/addons/submit/wizard.html

```diff
@@ -29,6 +29,8 @@ <h3>{{ _('Theme name') }}<span class="req" title="{{ _('required') }}">*</span><
       </ul>
         {% else %}
           <input type="text" id="theme-name"/>
+          <ul class="theme-name-error errorlist validator">
```

Comment:
on the screenshot the message(s) aren't on new lines so I think you need to tweak the css

Reply:
Do we need them on a new line?
I deliberately made them appear next to the field.

Reply:
Please, do. It's what we do elsewhere on the site - we don't know how long the error messages will be.